### PR TITLE
Fix the error on serialization/deserialization of classses in default package

### PR DESCRIPTION
**Pull request by moghaddam**
When a class is not in any package (it's in default package), the class.getPackage() method returns null and so the serialization/deserialization of such classes fails

### CODE REVIEW

**Comment by moghaddam** [comment]
Hi @m0mus 

What should I do to fix this ip-validation thing? I've never had this issue. As specified in the details page, seems I have to commit it using -s option. Should I repeat the steps on master branch and commit again with -s option?

**Comment by m0mus** [comment]
@moghaddam According to Eclipse Foundation rules only project official committers can commit. You are currently not an official committer. It means that we will review your merge request and commit it on your behalf.

**Comment by m0mus** [comment]
@moghaddam All classes must have a header with copyright. Please copy it from any other class. Put your name as a contributor.

**Comment by moghaddam** [comment]
OK @m0mus. I got it.
I just added the license on top of the two newly added file and used the Signed-off option on commit.

File: src/test/java/PackagelessClassTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
```
Comment (m0mus) [inline src/test/java/PackagelessClassTest.java]: Year has to be 2016.

File: src/test/java/PackagelessClassTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ ******************************************************************************/
+import org.junit.Test;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests the serialization/deserialization of a class that has no package
```
Comment (m0mus) [inline src/test/java/PackagelessClassTest.java]: We are always putting a dot "." at the end of the sentence in javadocs. I would like it to be consistent.

**Comment by m0mus** [comment]
@moghaddam I put some comments to the code itself. Just a couple of cosmetic changes. When you fix it you will get my commit approval.

**Comment by moghaddam** [comment]
@m0mus Sorry, I just fixed them ;)

File: src/main/java/org/eclipse/yasson/internal/AnnotationIntrospector.java [hunk 3]
```diff
@@ -412,7 +412,7 @@ private JsonbDateFormatter createJsonbDateFormatter(String format, String locale
      */
     public Optional<PropertyVisibilityStrategy> getPropertyVisibilityStrategy(Class<?> clazz) {
         JsonbVisibility visibilityAnnotation = findAnnotation(clazz.getDeclaredAnnotations(), JsonbVisibility.class);
-        if (visibilityAnnotation == null) {
+        if ((visibilityAnnotation == null) && (clazz.getPackage() != null)){
```
Comment (m0mus) [inline src/main/java/org/eclipse/yasson/internal/AnnotationIntrospector.java]: Please add a space before last "{"

File: src/test/java/PackagelessClassTest.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    @Test
+    public void testDeSerialization() throws Exception {
+        PackagelessModel packagelessClass = new PackagelessModel(12, "Hello World!");
+
+        String input = "{\"intValue\":12,\"stringValue\":\"Hello World!\"}";
+        PackagelessModel packagelessModel = jsonb.fromJson(input, PackagelessModel.class);
+        assertEquals(packagelessModel.getIntValue(), 12);
+        assertEquals(packagelessModel.getStringValue(), "Hello World!");
+    }
+
```
Comment (m0mus) [inline src/test/java/PackagelessClassTest.java]: Please remove this line.

Comment (moghaddam) [inline src/test/java/PackagelessClassTest.java]: fixed

**Comment by m0mus** [comment]
Sorry for getting too much into details. I usually do it this way. After several pull requests you will get use to it. :)

**Comment by moghaddam** [comment]
Yes, I like such consistency. What code formatting convention do you use/prefer? Eclise, Sun, .. . .?

**Comment by m0mus** [comment]
@moghaddam You are fast! I mean *really* fast! :)

**Pull request merged**